tests/fsharpqa/Source/InteractiveSession/Paket/PaketWithBrokenDepsFile/plot.fsx

```diff
@@ -1,3 +1,5 @@
+//<Expects status="Error" span="(5,6)" id="FS0039">The namespace or module 'XPlot' is not defined.</Expects>
```

Comment:
Not that we are so close to get this QA'ed, but should we plan for ill formatted paket directives errors?